src/overlays/actors/ovl_En_Syateki_Crow/z_en_syateki_crow.c

```diff
@@ -292,9 +290,9 @@ s32 EnSyatekiCrow_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx*
                                    Actor* thisx) {
     EnSyatekiCrow* this = THIS;
 
-    if (limbIndex == 7) {
+    if (limbIndex == OBJECT_CROW_LIMB_07) {
         rot->y += (s16)(3072.0f * sin_rad(this->skelAnime.curFrame * 0.7853982f));
-    } else if (limbIndex == 8) {
+    } else if (limbIndex == OBJECT_CROW_LIMB_08) {
         rot->y += (s16)(5120.0f * sin_rad((this->skelAnime.curFrame + 2.5f) * 0.7853982f));
```

Comment:
These two floats `0.7853982`
Is that M_PI/4 ?

Reply:
Yep